airflow/contrib/operators/bigquery_get_data.py

```diff
@@ -78,6 +80,7 @@ def __init__(self,
                  selected_fields=None,
                  bigquery_conn_id='bigquery_default',
                  delegate_to=None,
+                 do_xcom_push=True,
```

Comment:
`True` is default behaviour, so then we need to add a line to the `Updating.md`

Reply:
Personally I think `False` is making more sense. Was this discussed by other folks?

In addition, if that's the case and we go for `True`, other changes would be needed, for example, the default value of `xcom_push` in [`BashOperator`](https://github.com/apache/incubator-airflow/blob/master/airflow/operators/bash_operator.py) is `False` currently.

Reply:
Did another quick check: seems default value of most existing `xcom_push`/`do_xcom_push` arguments are `False`. We should make all of them consistent no matter we decide to use `True` or `False`.

BTW, now I understand why @itscaro put `do_xcom_push` as the variable name since it is used in some of the operators. This may cause confusion.

Reply:
@XD-DENG I fully agree. It would be even nicer to add the logic somewhere around here: https://github.com/apache/incubator-airflow/blob/b85210d8596c1e05fbf3da2941aaca8170ae871e/airflow/models.py#L1643

Instead of duplicating it in every operator. This makes it very easy for newcomers to forget about xcom.

Reply:
@XD-DENG Thanks for checking, lets go for `False` then. It makes also more sense to me since it might clutter up the Database quickly. We should still add a line to `Updating.md` and target the fix for Airflow 2.0